beps/0005-definitive-plugin-listing/README.md

```diff
@@ -166,21 +166,18 @@ interface BackstageInstance {
   };
 }
 
-interface DeploymentMetadataPluginExtensionPoint {
-  setInstances: (instances: BackstageInstance[]) => Promise<void>;
+interface DeploymentMetadataService {
+  listInstances: () => BackstageInstance[];
 }
 ```
 
-By default, this will either
+By default, this will
 a. be the exact same as `InstanceMetadataService` if we are not using `HostDiscovery` static config, or
-b. TODO: How will users define their endpoints statically such that we can access other instance's `InstanceMetadataService` HTTP APIs?
+b. use the `HostDiscovery` static config to fetch other instance's base URLs.
```

Comment:
another option is to simply rely on local + explicitly configured plugins in discovery for this for now